Dockerfile

```
@@ -57,7 +42,19 @@ RUN set -eux ; \
     && ln -sf /usr/bin/python3 /usr/bin/python
 RUN sudo python3 -m pip install --upgrade pip
 
+# CSI / k8s / fuse / goofys dependency
 COPY --from=go /go/bin/csc /usr/bin/csc
+# S3 FUSE support
+RUN set -eux ; \
+    ARCH="$(arch)"; \
+    case "${ARCH}" in \
+        x86_64)  arch='x86_64' ;; \
+        aarch64) arch='arm64' ;; \
+        *) echo "Unsupported architecture: ${ARCH}"; exit 1 ;; \
+    esac; \
+    curl -L -o /tmp/mount-s3.rpm "https://s3.amazonaws.com/mountpoint-s3-release/latest/${arch}/mount-s3.rpm"; \
+    dnf install -y /tmp/mount-s3.rpm; \
+    rm -f /tmp/mount-s3.rpm
```

Suggest to use a fixed version:
```suggestion

RUN set -eux ; \
    ARCH="$(arch)"; \
    MOUNTPOINT_S3_VERSION="1.19.0"; \
    case "${ARCH}" in \
        x86_64)  url="https://s3.amazonaws.com/mountpoint-s3-release/${MOUNTPOINT_S3_VERSION}/x86_64/mount-s3-${MOUNTPOINT_S3_VERSION}-x86_64.rpm" ;; \
        aarch64) url="https://s3.amazonaws.com/mountpoint-s3-release/${MOUNTPOINT_S3_VERSION}/arm64/mount-s3-${MOUNTPOINT_S3_VERSION}-arm64.rpm" ;; \
        *) echo "Unsupported architecture: ${ARCH}"; exit 1 ;; \
    esac; \
    curl -L ${url} -o mount-s3.rpm ; \
    dnf install -y mount-s3.rpm ; \
    rm -f mount-s3.rpm
```

---

Thanks @jojochuang , it's a better solution.
I will modify it ASAP.